go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/app-sre/go-qontract-reconcile
 
-go 1.18
+go 1.19
 
 require (
 	github.com/Khan/genqlient v0.5.0
-	github.com/ProtonMail/gopenpgp/v2 v2.4.10
+	github.com/ProtonMail/gopenpgp/v2 v2.1.10
```

Comment:
**To the reviewer:**

This is a by-product of the openpgp package from the go-crypto suite being pinned to a specific, and in this particular case, older version.

Given that we aim to keep the version, and thus features, parity with Terraform, this is acceptable and should have no severe side-effects.